SPECS/coreutils/coreutils.spec

```diff
@@ -60,20 +60,16 @@ make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-install -vdm 755 %{buildroot}/bin
 install -vdm 755 %{buildroot}%{_sbindir}
 install -vdm 755 %{buildroot}%{_mandir}/man8
-mv -v %{buildroot}%{_bindir}/{arch,cat,chgrp,chmod,chown,cp,date,dd,df,echo} %{buildroot}/bin
-mv -v %{buildroot}%{_bindir}/{false,ln,ls,mkdir,mknod,mv,pwd,rm} %{buildroot}/bin
-mv -v %{buildroot}%{_bindir}/{rmdir,stty,sync,true,uname,test,[} %{buildroot}/bin
 mv -v %{buildroot}%{_bindir}/chroot %{buildroot}%{_sbindir}
 mv -v %{buildroot}%{_mandir}/man1/chroot.1 %{buildroot}%{_mandir}/man8/chroot.8
 sed -i s/\"1\"/\"8\"/1 %{buildroot}%{_mandir}/man8/chroot.8
-mv -v %{buildroot}%{_bindir}/{head,sleep,nice} %{buildroot}/bin
 rm -rf %{buildroot}%{_infodir}
 install -vdm755 %{buildroot}%{_sysconfdir}/profile.d
 install -m 0644 %{SOURCE1} %{buildroot}%{_sysconfdir}/profile.d/
 %find_lang %{name}
+ln -s %{buildroot}%{_bindir} %{buildroot}/
```

Comment:
Does this create a symlink in / or in /bin? Thanks

Reply:
+1 to this question. I am also not sure how this symlink is supposed to function. Where does the symlink file ultimately end up? And it looks to be referencing `%{buildroot}%{_bindir}` which wouldn't exist of the final image where this gets installed to, right?

Reply:
it creates a symlink in under coreutils  package, so under coreutils it looks like /bin ->> /usr/bin